syntax = "proto3";

option go_package = "./spec";

service Ats {
  rpc SendStatus(SendStatusRequest) returns (SendStatusResponse) {}
}

message SendStatusRequest {
  enum SensorName {
    unknown = 0;

    shinjuku_d1 = 1;
    shinjuku_d2 = 2;

    sakurajosui_d1 = 11;
    sakurajosui_d2 = 12;
    sakurajosui_d3 = 13;
    sakurajosui_d4 = 14;

    chofu_d1 = 21;
    chofu_d2 = 22;
    chofu_d3 = 23;
    chofu_d4 = 24;
    chofu_d5 = 25;
    chofu_d6 = 26;

    hashimoto_d1 = 31;
    hashimoto_d2 = 32;

    hachioji_d1 = 41;
    hachioji_d2 = 42;

    wakabadai_d1 = 51;
    wakabadai_d2 = 52;
  }
  SensorName sensor = 1;
}
message SendStatusResponse {
  enum Response {
    UNKNOWN = 0;
    SUCCESS = 1;
    FAILED = 2;
  }
  Response response = 1;
}
